src/app/events/event-detail/event-detail.page.ts

```diff
@@ -21,45 +22,29 @@ export class EventDetailPage implements OnInit {
   defaultHref: string = '/tabs/events';
   backButton: boolean = true;
 
-  mockEvent: Event = {
-    shortTitle: 'Кондофрей Драг 2023',
-    longTitle: '"Кондофрей Драг 2023" - село Кондофрей, летище "София-Запад"',
-    imageUrl: 'https://i.ytimg.com/vi/t52ovr-qdB0/maxresdefault.jpg',
-    shortDescription: 'Драг Рейсинг - Кондофрей [29-30 Април]',
-    longDescription: `Кондофрей Драг е най-голямото и вълнуващо драг състезание в България, което привлича състезатели и фенове от цялата страна и дори извън нея. Събитието включва разнообразие от класове, от улични автомобили до професионални драгстери. Зрителите могат да се насладят на страхотното състезание, както и на храна и напитки. Гледайте как мощни машини ускоряват от 0 до 60 за секунди и достигат скорости от над 200км/ч.`,
-    dates: [
-      { date: '15 декември 2023', startTime: '9:00', endTime: '19:30' },
-      { date: '16 декември 2023', startTime: '10:00', endTime: '16:30' },
-    ],
+  eventId: string = '';
+
+  event: Event = {
```

Comment:
Why initialized with all properties?

Reply:
It was needed because I wanted to use the Event interface. Changed to any and added only the needed properties so not to throw an error before the event is loaded